### DISCUSSION TITLE
[Script request] ERPNext

### DISCUSSION THREAD

**Discussion by Feramance** [community-scripts/ProxmoxVE#52] [Request script]
### Application Name

ERPNext

### Website

https://erpnext.com/

### Description

ERPNext is the world's best free and open source ERP

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by primeEDV** [comment]
Hello, I'm interested in a script that can install the open source version of erpnext and provide the full functionality by redirecting erpnext.org to the .com.

I'm looking forward to further information.

**Comment by JamesonRGrieve** [comment]
I'm working on a pull request for this right now. Hoping to offer load balancing LXCs similar to their docker configuration but at minimum planning to get at least an all-in-one out the door.

**Reply by WWE-Corey** [reply]
Thank you - not trying to leave an unsubstantial remark but I struggle with updating ERPNext each year and would love and appreciate a more simple method to install and ideally upgrade a running instance.

**Reply by tremor021** [reply]
Open source is like that. Applications change all the time, breaking changes are not uncommon.

Even when we make a update script, the change in the way the app works breaks that. So, always make snapshots before updating

**Reply by WWE-Corey** [reply]
Found this today but have not tested: https://github.com/Marcello1099/proxmox-erpnext

**Comment by MickLesk** [comment]
https://github.com/community-scripts/ProxmoxVED/issues/1616
